### ISSUE TITLE
How to trace errors from "didEncounterErrors"?

### ISSUE THREAD

**Issue by StarpTech** [DanielMSchmidt/apollo-opentracing#438]
For example when a field can't be mapped to the GraphQL type

```
{"level":50,"time":1625050489069,"pid":32066,"hostname":"foo","name":"GraphQLError","code":"GRAPHQL_ERR","message":"Cannot return null for non-nullable field Teaser.id."...
```

This error is only logged by `didEncounterErrors`. Finding the right span is very fuzzy and attaching an `error=true` tag has no result.

https://www.apollographql.com/docs/apollo-server/integrations/plugins/#request-lifecycle-event-flow

We are using Jaeger.

**Comment by leeweisberger** [comment]
Opened a pr here: https://github.com/DanielMSchmidt/apollo-opentracing/pull/439

**Comment by StarpTech** [comment]
Fix with #439
